### ISSUE TITLE
Export Not Functioning

### ISSUE THREAD

**Issue by jtg-pics** [gs-gs/staruml-model-interchange#37]
Steps to reproduce:
1. Create a new project (File > New)
2. Attempt to export (Tools > model interchange > export)
    Select the top level package ("Untitled")
    OK

Error shown:
![image](https://user-images.githubusercontent.com/63125232/80746237-8a47ce80-8aef-11ea-8a25-81474f946290.png)

Additional information:
Using StarUML version 3.2.2

**Comment by faizanvahevaria** [comment]
@jtg-pics 
Element you have selected is StarUML project so to export you must select package in StarUML

Please close if this work

**Comment by jtg-pics** [comment]
@faizanvahevaria Thanks, looks like StarUML doesn't create any packages as part of the default structure. After manually creating a package I was able to generate a .json file however the file contains no meaningful model information:
![image](https://user-images.githubusercontent.com/63125232/80816239-24b21b80-8b9d-11ea-883e-bbaa9ab34bbb.png)

**Comment by faizanvahevaria** [comment]
@jtg-pics 
Can your please send me your mdj file

**Comment by jtg-pics** [comment]
https://drive.google.com/open?id=1gKeYN8w3GNbvf8Rg1fD1TVisJv4NTFVn

**Comment by faizanvahevaria** [comment]
@jtg-pics 
The file is not same as shown in https://github.com/gs-gs/staruml-model-interchange/issues/37#issuecomment-622429131

Though i have made changes in your mdj file and exported json file corresponding that file.

See updated mdj file and generated json file attached zip file
[mdj_and_json.zip](https://github.com/gs-gs/staruml-model-interchange/files/4575242/mdj_and_json.zip)

And for this extension export/import 
this JSONSchema structure should be followed -> [Link](https://github.com/gs-gs/staruml-model-interchange/blob/master/docs/domain_sample_jsoncschema.json)
in StarUML project

**Comment by jtg-pics** [comment]
Didn't save that project. That file is experiencing the same issue, confirmed before sharing.

**Comment by faizanvahevaria** [comment]
Try downloading from your link https://github.com/gs-gs/staruml-model-interchange/issues/37#issuecomment-623460939
Its not same as you have shown in the screenshot
https://user-images.githubusercontent.com/63125232/80816239-24b21b80-8b9d-11ea-883e-bbaa9ab34bbb.png

**Comment by jtg-pics** [comment]
Correct. I no longer have the file shown in the screenshot. The file I have shared experiences the same exact error as the one I was originally working with that is shown in the screenshot.

**Comment by faizanvahevaria** [comment]
So,
model-interchange export follows the project structure as mentioned in this ticket : [link](https://github.com/gs-gs/staruml-model-interchange/issues/21)

you can find sample mdj file and exported json file in the attachment
[sample_mdj_and_json.zip](https://github.com/gs-gs/staruml-model-interchange/files/4575436/sample_mdj_and_json.zip)
